src/pages/patientView/PatientViewPage.tsx

```diff
@@ -257,6 +276,19 @@ export default class PatientViewPage extends React.Component<
         );
     }
 
+    private shouldShowResources(
```

Comment:
why not computed getter?

Reply:
just following the conventions. but they should all be computed actually